src/app/beer_garden/db/mongo/pruner.py

```diff
@@ -30,7 +32,17 @@ def add_task(
             }
         )
 
+    def cancel_in_progress(self):
+        in_progress = Request.objects(Q2(status="IN_PROGRESS") | Q2(status="CREATED"))
```

Comment:
You don't need to use `Q` to construct this query. You can just do:

`Request.objects.filter(status__in=["IN_PROGRESS", "CREATED"])`